Sources/MapboxMaps/Ornaments/ScaleBar/MapboxScaleBarOrnamentView.swift

```diff
@@ -343,19 +349,23 @@ internal class MapboxScaleBarOrnamentView: UIView {
     }
 
     private func updateVisibility() {
-        let maximumDistance: CLLocationDistance = Double(maximumWidth) * unitsPerPoint
-        let allowedDistance = isMetricLocale ?
-                              Constants.metricTable.last!.distance : Constants.imperialTable.last!.distance
-        let alpha: CGFloat = maximumDistance > allowedDistance ? 0 : 1
-
-        if alpha != self.alpha {
-            UIView.animate(withDuration: 0.2,
-                           delay: 0,
-                           options: .beginFromCurrentState,
-                           animations: {
-                            self.alpha = alpha
-            },
-                           completion: nil)
+        if visibility == .hidden {
+            self.alpha = 0
```

Comment:
`MapboxScaleBarOrnamentView` is of type `UIView`. Use the `isHidden` property instead